frontend/server/src/Authorization.php

```diff
@@ -217,6 +217,18 @@ public static function canEditClarification(
                 )));
     }
 
+    public static function isUnderThirteenUser(\OmegaUp\DAO\VO\Users $user): bool {
+        // This is mostly for users who hasn't give us their birth day
+        if (is_null($user->birth_date)) {
+            return false;
+        }
+        // User's age is U13? $user->birth_date - current date then return true, otherwise return false
+        return $user->birth_date < strtotime(
```

Comment:
The type of the field `birth_date` in the table `Users` is a Date (not a timestamp), so we need to convert it to a valid date in PHP
```suggestion
        return strtotime($user->birth_date) >= strtotime(
```